tests/e2e/SMS/TelnyxTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests\E2E;
+
+use Utopia\Messaging\Adapters\SMS\Telnyx;
+use Utopia\Messaging\Messages\SMS;
+
+class TelnyxTest extends Base
+{
+    /**
+     * @throws \Exception
+     */
+    public function testSendSMS()
+    {
+        // $sender = new Telnyx(getenv('TELNYX_API_KEY'));
+
+        // $message = new SMS(
+        //     to: ['+18034041123'],
+        //     content: 'Test Content',
+        //     from: '+15005550006'
+        // );
+
+        // $result = \json_decode($sender->send($message), true);
+
+        // $this->assertEquals('success', $result["type"]);
+
+        $this->markTestSkipped('Telnyx had no testing numbers available at this time.');
```

Comment:
Has this changed?

Reply:
Ping

Reply:
no, still showing that I don't have a testing number available